packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx

```diff
@@ -196,6 +230,8 @@ export class FbFieldDropdownItem {
       return null;
     }
 
+    const objMaxLimits = getMaximumLimitsConfig(this.productName);
```

Comment:
@Srikannan-E Can you please use getMaxLimitProperty and get the count?

Reply:
I done the changes bro